internal/cluster/distribution/eks/eksadapter/node_pool_manager.go

```diff
@@ -94,3 +115,95 @@ func (n nodePoolManager) UpdateNodePool(
 func generateNodePoolStackName(clusterName string, poolName string) string {
 	return "pipeline-eks-nodepool-" + clusterName + "-" + poolName
 }
+
+// ListNodePools is for listing node pools from CloudsetFormation and NodePoolLabelSets
```

Comment:
```suggestion
// ListNodePools lists node pools.
```

Reply:
Any chance we could write some unit tests for this? Using kube fake client and some sort of mock for the cf client? Unit test would be a great start, functional tests would be harder now.

Reply:
Fixed in 1427f4df2964a33261074d8ebcc49a6ecf91b105.